### ISSUE TITLE
Since 9.1, the "Use Windows File Explorer sort order" setting is no longer observed

### ISSUE THREAD

**Issue by PancakeProphet** [d2phap/ImageGlass#1943]
### System information
- Windows OS version: 11 23H2
- ImageGlass version: 9.1.6.14
- ImageGlass release: Classic

### To Reproduce
1. In ImageGlass, in Settings > Image > Image Loading, set "Image Loading order" to "Name (default)" & "Ascending", and check "Use Windows File Explorer sort order if possible".
2. In Windows Explorer, sort a folder containing several images by Date, Descending (right-click > Sort By).
3. Open the **first** image in the list so that it launches in ImageGlass.

### Actual behavior
- In ImageGlass, the load order of the images doesn't respect the "Use Windows File Explorer sort order if possible" setting. Instead, the order used is the "Image Loading order".
- In other words: if a Windows Explorer folder is sorted by Date and ImageGlass has an Image Loading order by Name, when opening an image from Windows Explorer, ImageGlass's loading order will be by Name instead of by Date.
- The behavior is a **regression**: the expected behavior works as expected in ImageGlass 9.0.11.502.

### Expected behavior
- In ImageGlass, when the "Use Windows File Explorer sort order if possible" setting is enabled and an image is opened from Windows Explorer, the Image Loading order of the images should respect the sort order in the Windows Explorer folder containing the launched image.

**Comment by d2phap** [comment]
Duplicated of #544.

ImageGlass 9.1 has limitation for the "Use Windows File Explorer sort order" feature. It only supports sort orders: [Name, Extension, File Size, EXIF Rating](https://imageglass.org/docs/features#limitations). That's why the option contains "if possible".

Good news for you that, recently I have done long research to completely add support for any sort order from File Explorer including Explorer's Search #404. You need to wait for v9.2 (soonest) to be able to use it.
